### DISCUSSION TITLE
Report a RiMusic clone in play store

### DISCUSSION THREAD

**Discussion by aneesh1122** [fast4x/RiMusic#3671] [General]
https://play.google.com/store/apps/details?id=com.music.downloader.sptfifst

Please report this app for Impersonation

This app's dev stole RiMusic's code and put ads on the app.

**Comment by NOBELLIST** [comment]
Tried it and my goodess whyis the rating 4.8

**Reply by aneesh1122** [reply]
And 1 lac downloads. I don't know how to take it down

**Reply by NOBELLIST** [reply]
Maybe spam report? Google plays report system is just bad.

**Reply by aneesh1122** [reply]
> Maybe spam report? Google plays report system is just bad. 

I reported it but as usual haven't heard anything back from google
